sites/partners/cypress/support/commands.js

```diff
@@ -475,3 +475,14 @@ Cypress.Commands.add("addMinimalApplication", (listingName) => {
   })
   cy.getByID("submitApplicationButton").click()
 })
+
+Cypress.Commands.add("findAndOpenListing", (listingName) => {
+  cy.visit("/")
+  cy.contains("Listings")
+  cy.getByTestId("ag-search-input").should("be.visible").type(listingName)
+  cy.getByTestId(listingName).first().click()
+})
+
+Cypress.Keyboard.defaults({
+  keystrokeDelay: 1,
+})
```

Comment:
Will this slow every test down by 1 millisecond * number of key strokes? I don't think we have enough keystrokes to make that matter, just curious

Reply:
I ended up removing this, but the default is 10, so it would speed things up